src/amber/cli/templates/app/shard.yml.ecr

```diff
@@ -57,3 +57,7 @@ development_dependencies:
   garnet_spec:
     github: amberframework/garnet-spec
     version: ~> 0.1.1
+
```

Comment:
If Amber requires this as a dependency, do we need it here as well?

Reply:
You're right, let me fix this 👍